### PR TITLE
Add live url to the readme.

### PR DESCRIPTION
**Pull request by empeje**
Add https://picnic.tech/picnic-lunch-and-learn/ to the readme
Solves #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Lunch and Learn website
 
+## Live Site
+
+You can see the Lunch and Learn session schedule [here][live-url].
```
Comment (nov1n) [inline README.md]:
```suggestion
The Lunch & Learn website is available at [picnic.tech/picnic-lunch-and-learn][live-url].
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Lunch and Learn website
 
+## Live Site
```
Comment (nov1n) [inline README.md]:
```suggestion
## Live Website
```

File: README.md [hunk 3]
```diff
@@ -9,3 +13,5 @@ Make changes and commit them to `master`. Then commit and push.
 To deploy your changes, run:
 
 `yarn run deploy`
+
+[live-url]: https://picnic.tech/picnic-lunch-and-learn/
```
Comment (nov1n) [inline README.md]: Maybe we can make `live-url` a bit more descriptive?

**Review by nov1n — COMMENTED** [review]
Thanks for contributing 💯

File: README.md (hunk 3 above)
Comment (empeje) [reply README.md]: Updated @nov1n

Comment (nov1n) [reply README.md]: Looks good 👍

**Comment by nov1n** [comment]
Thanks again 🎉

**Pull request merged**